lib/kitchen/directions/bake_generic_eoc_section/v1.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Kitchen::Directions::BakeGenericEocSection
+  class V1
+    renderable
+
+    def bake(chapter:, metadata_source:, klass:, append_to: nil, uuid_prefix: '.', content_transform: [])
+      @klass = klass
+      @metadata = metadata_source.children_to_keep.copy
+      @title = I18n.t(:"eoc.#{klass}")
+      @uuid_prefix = uuid_prefix
+
+      section_clipboard = Kitchen::Clipboard.new
+
+      chapter.non_introduction_pages.each do |page|
+        sections = page.search("section.#{klass}")
+        # Transforms the content of the section according to the given strategies, if given
+        content_transform.each do |strat|
+          sections.each do |section|
+            Kitchen::Directions::EocContentTransform.v1(section: section, strategy: strat)
```

Comment:
What do y'all think of this way of addressing content changes to EoC sections? There are a couple of different ways of invoking methods in ruby, and I chose to create a mini-direction, `EocContentTransform`, whose methods perform small modifications to `<section>` elements. But I think this could be done in other ways too. Thoughts?

Reply:
Hm, while most EOC content uses`<section>` elements, my book has at least two that use something else (Glossary and Solutions).  Would it be alright to have a slightly broader strategy?

Reply:
sure! I can adjust this so that any element can be passed in